civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -15,25 +16,36 @@
 
 def initialize_notebook_from_platform(notebook_path):
     """ This runs on startup to initialize the notebook """
+    logger.info('Getting URL for notebook file')
```

Comment:
maybe more accurate to say `retrieving notebook info from platform` or something, since this also at least contains the requirements url